pytorch_lightning/lite/wrappers.py

```diff
@@ -110,49 +106,6 @@ def _convert_float_tensor(t: Tensor) -> Tensor:
         return output
 
 
-def _wrap_init(init: Callable) -> Callable:
```

Comment:
Reviewer: These three functions moved to utilities/data.py and are now shared between Lightning and Lite